docs/installation/installation_google_cloud_kubernetes.md

```diff
@@ -0,0 +1,47 @@
+## Installing Yacy on Google Cloud with Docker

```

Comment:
@niranjan94 I wrote the documentation on both ways using jubernetes and only docker, this is an older state of pull, updating it now.